src/PlaywrightSharp/Waiter.cs

```diff
@@ -71,6 +71,35 @@ internal Task<T> WaitForEventAsync<T>(object eventSource, string e, Func<T, bool
             return WaitForPromiseAsync(task, dispose);
         }
 
+        internal Task WaitForEventAsync(object eventSource, string e)
```

Comment:
This looks like we'll do some consolidation after both PRs merge - https://github.com/microsoft/playwright-sharp/pull/1263/files#diff-9c8cb0708a885729f8dc5b88825599418e9b5042c2989b0ffd8a9727574ce9f6

Reply:
I copied from there. I will be the same code when both are merged.